_docs/portal/1.200.0.md

```diff
@@ -0,0 +1,17 @@
+---
+layout: page
+title: 1.200.0
+permalink: /docs/portal/1.200.0/
+parent: Portal
+nav_order: -10001
+release_date: 15.07.2024
+---
+
+## Import users accepts e-mail addresses with capital letters
+Import users from .csv file should now accept e-mail addresses with capital letters. Previously, such users were filtered out.
+
+## Issue with lock state not being updated on device details page
+Lock's state should be updated both in title and in device details section at least every 10 seconds.
```

Comment:
Instead of using passive use active.  Portal automatically updates the device list and details page with the latest lock state at least every 10 seconds.